ublox_derive/src/output/gen_recv_code.rs

```diff
@@ -68,12 +69,17 @@ pub fn generate_recv_code_for_packet(dbg_ctx: DebugContext, pack_descr: &PackDes
                 &self.0
             }
 
+            pub fn to_ref(&self) -> #ref_name {
+                self.into()
+            }
+
```

Comment:
Should be an implementation of [AsRef](https://doc.rust-lang.org/std/convert/trait.AsRef.html) instead

```suggestion
```

Reply:
well that remark could apply to existing structures as well..
I'm able to implement a referecing method like this, but unable to implement AsRef which seems more complex to implement